src/main/java/com/snowflake/kafka/connector/internal/SnowflakeErrors.java

```diff
@@ -414,7 +414,7 @@ public SnowflakeKafkaConnectorException getException(
   {
     if(telemetryService != null)
     {
-      telemetryService.reportKafkaFatalError(getCode());
+      telemetryService.reportKafkaFatalError(getCode() + msg);
```

Comment:
Is there a limit on the size of msg

Reply:
No. Should I truncate the msg to the first 500 characters? The first several hundreds characters should contains enough information for debugging.